.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: node_js
 node_js:
-  - "11"
-  - "10"
-  - "8"
+  - "16.15.0"
```

Comment:
This fixes the Travis errors that had prevented all PRs from merging. We're now only checking this on LTS